tests/Feature/Api/V1Test.php

```diff
@@ -178,23 +178,88 @@ public function testGetAchievementOfTheWeek(): void
             ]);
     }
 
-    // public function testGetAchievementsEarnedBetween(): void
-    // {
-    //     // TODO
-    //
-    //     $this->get($this->apiUrl('GetAchievementsEarnedBetween'))
-    //         ->assertSuccessful()
-    //         ->assertExactJson([]);
-    // }
-    //
-    // public function testGetAchievementsEarnedOnDay(): void
-    // {
-    //     // TODO
-    //
-    //     $this->get($this->apiUrl('GetAchievementsEarnedOnDay'))
-    //         ->assertSuccessful()
-    //         ->assertExactJson([]);
-    // }
+    public function testGetAchievementsEarnedBetweenEmptyResponse(): void
+    {
+        $this->get($this->apiUrl('GetAchievementsEarnedBetween'))
+            ->assertSuccessful()
+            ->assertExactJson([]);
+    }
+
+    public function testGetAchievementsEarnedBetween(): void
+    {
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->ID]);
+        /** @var Achievement $achievement */
+        $achievement = Achievement::factory()->published()->create(['GameID' => $game->ID, 'Points' => 100]);
+        /** @var PlayerAchievement $unlock */
+        $unlock = PlayerAchievement::factory()->create(['AchievementID' => $achievement->ID, 'User' => $this->user->User]);
+
+        $this->get(
+            $this->apiUrl('GetAchievementsEarnedBetween', [
+                'u' => $this->user->User,
+                'f' => Carbon::now()->subDay()->startOfDay()->unix(),
+                't' => Carbon::now()->addDay()->endOfDay()->unix(),
+            ])
+        )
```

Comment:
Apparently this comment didn't get saved:

Suggest having additional unlocks outside of the filter range to validate the filter. Similarly, have additional unlocks in the filter range for a different user and Unofficial.

Should also have at least one more unlock inside the filter range to validate `CumulScore`. Could also consider using a different game/console for the additional unlock to ensure distinctive related data is returned.

Reply:
I want to keep it simple for now - I welcome any improvements to the test coverage. For now, I want to be done with this.

Reply:
Improvements to the tests which will increase coverage will follow. Happy paths only for now to set the baseline.